internal/tagging/tagging_test.go

```diff
@@ -0,0 +1,71 @@
+package tagging
+
+import (
+	"testing"
+
+	"github.com/env0/terratag/internal/common"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTagBlock_MergeOrder(t *testing.T) {
```

Comment:
Decided to add it as unit test.
much cleaner than running an integration test.
I tested it manually to ensure it's working as expected from a terraform/tofu perspective.

Reply:
We talked, it will be a lot of work since very specific and UT should be enough